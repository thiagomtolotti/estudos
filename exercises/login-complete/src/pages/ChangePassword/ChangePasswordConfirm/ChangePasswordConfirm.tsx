import Link from "next/link";

export const ChangePasswordConfirm = () => {
	return (
		<div>
			<p>
				Sua senha foi alterada corretamente, agora é só fazer o login.
			</p>

			<Link href={"/login"} data-testid="login-button">
				<button>Voltar ao login</button>
			</Link>
		</div>
	);
};
