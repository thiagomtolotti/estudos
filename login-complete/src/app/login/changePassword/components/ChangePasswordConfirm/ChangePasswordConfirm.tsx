export const ChangePasswordConfirm = () => {
	return <h2>auhsdhudsh</h2>;
};
