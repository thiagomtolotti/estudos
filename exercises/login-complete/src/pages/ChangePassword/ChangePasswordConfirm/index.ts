import { ChangePasswordConfirm } from "./ChangePasswordConfirm";

export default ChangePasswordConfirm;
