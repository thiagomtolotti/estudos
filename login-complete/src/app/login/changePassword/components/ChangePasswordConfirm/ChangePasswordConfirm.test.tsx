import { RenderResult, render } from "@testing-library/react";
import ChangePasswordConfirm from ".";

describe("Change Password Confirm", () => {
	let renderResult: RenderResult;

	beforeEach(() => {
		renderResult = render(<ChangePasswordConfirm />);
	});

	it("Should render correctly", () => {
		expect(renderResult.container).toMatchSnapshot();
	});
});
