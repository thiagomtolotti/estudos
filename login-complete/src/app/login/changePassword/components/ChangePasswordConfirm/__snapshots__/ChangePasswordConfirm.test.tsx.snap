// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Change Password Confirm Should render correctly 1`] = `
<div>
  <h2>
    auhsdhudsh
  </h2>
</div>
`;
